R/Flood_CamML_App/app.R

```diff
@@ -288,85 +288,548 @@ ui <- material_page(
+  mirlo_predict_unsupervised <- reactive({
+    # code to predict. Can use "Mirlo.jpg" as path for keras code
+  })
+  northdock_predict_unsupervised <- reactive({
+    # code to predict. 
+  })
+  southdock_predict_unsupervised <- reactive({
+    # code to predict. 
+  })
+  southocracoke_predict_unsupervised <- reactive({
+    # code to predict. 
+  })
+  
+  
+  #--------------- Display Camera Feeds ----------------------
+  
+  
+  # Initialize wait screen for individual pics. Need one of these for each site
+  w <- Waiter$new(id = "mirlo_selection_unsupervised",
+                  html = spin_3k(),
+                  color = transparent(.75))
+  
+  
+  # Render camera images
+  output$mirlo_picture_unsupervised <- render_cam_image(cam_name = "Mirlo", alt_name = "Mirlo Beach")
+  output$northdock_picture_unsupervised <- render_cam_image(cam_name = "NorthDock", alt_name = "NorthDock")
+  output$southdock_picture_unsupervised <- render_cam_image(cam_name = "SouthDock", alt_name = "SouthDock")
+  output$southocracoke_picture_unsupervised <- render_cam_image(cam_name = "SouthOcracoke", alt_name = "SouthOcracoke")
+
+  
+  
+  #--------------- Camera Feedback UI ----------------------
+  
+  # Display model classification around pictures
+  # Apply model prediction and radio buttons to each camera
+  output$mirlo_selection_unsupervised <- render_camera_ui(cam_name = "Mirlo", 
+                                                          cam_time = mirlo_time_reactive_unsupervised(),
+                                                          model_prediction = mirlo_predict_unsupervised(),
+                                                          id_suffix = "_unsupervised")
+  output$northdock_selection_unsupervised <- render_camera_ui(cam_name = "NorthDock", 
+                                                              cam_time = northdock_time_reactive_unsupervised(),
+                                                              model_prediction = northdock_predict_unsupervised(),
+                                                              id_suffix = "_unsupervised")
+  output$southdock_selection_unsupervised <- render_camera_ui(cam_name = "SouthDock", 
+                                                              cam_time = southdock_time_reactive_unsupervised(),
+                                                              model_prediction = southdock_predict_unsupervised(),
+                                                              id_suffix = "_unsupervised")
+  output$southocracoke_selection_unsupervised <- render_camera_ui(cam_name = "SouthOcracoke", 
+                                                                  cam_time = southocracoke_time_reactive_unsupervised(),
+                                                                  model_prediction = southocracoke_predict_unsupervised(),
+                                                                  id_suffix = "_unsupervised")
+  
+  
```

Comment:
optional id_suffix adds "_unsupervised" to the input and output tags that downstream things depend on.